test/unit/property-list.test.js

```diff
@@ -960,6 +963,24 @@ describe('PropertyList', function () {
                 val: 'value2'
             }]);
         });
+
+        it('should throw error when unable to upsert into a list Type that does not support .update()', function () {
+            FakeType.prototype.update = null;
```

Comment:
Don't rely on the global variable.

Reply:
Created a local variable for `FakeType`.